src/components/ApplicationQuestionsList/ApplicationQuestionsList.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, useCallback } from "react";
+
+import Caption from "@govuk-react/caption";
+import { Link } from "@reach/router";
+import Table from "@govuk-react/table";
+import Button from "@govuk-react/button";
+
+import { LinkButton } from "../../theme";
+import { UpdateApplicationQuestionInput } from "../../API";
+
+interface Props {
+    applicationId: string;
+    questions: (UpdateApplicationQuestionInput | null)[] | null;
+    addQuestion: (id: string) => void;
+    deleteQuestion: (id: string) => void;
+}
+
+export const ApplicationQuestionsList: FC<Props> = ({
+    questions,
+    applicationId,
+    addQuestion,
+    deleteQuestion
+}) => {
+    return (
+        <>
+            <Caption> Application questions </Caption>
+            <Table>
+                {questions &&
+                    questions.map(
+                        (question: UpdateApplicationQuestionInput | null) => {
```

Comment:
You shouldn't need to type this: it should be able to infer the type from the type of `questions`. Is it complaining if you leave it out?